src/components/hostNav.jsx

```diff
@@ -1,17 +1,19 @@
 import { Link } from "react-router-dom";
-import { useLocation } from "react-router-dom";
+import { useLocation, useParams } from "react-router-dom";
 
 export default function HostNav() {
   const location = useLocation();
+  const vanId = useParams();
+
   return (
     <div className="flex gap-4 font-bold bg-[#fff7ed] px-4 py-8 text-[13px]">
       <Link
         className={`${
-          location.pathname === "/host/dashboard"
+          location.pathname === "/host"
             ? "underline-offset-2 underline"
             : "text-[#4d4d4d]"
         }`}
-        to="/host/dashboard"
+        to="/host"
       >
```

Comment:
You can use NavLink component to achieve the same result rather than using the location object directly.